Backend/tests/notTest_algebra.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import sys
+from os import path
+import ConfigParser as cp
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
+from problems import mathstepsWrapper
+
+
+#Configuration setup
+configPath = "/config/prodConfig.ini"
+parser = cp.ConfigParser()
+parser.read(configPath)
+
+mathstepsPath = parser.get("MathstepsLocation", 'path')
+
+class MentiiMathstepsTests(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(self):
+    mathstepsWrapper.setMathstepsLocation(mathstepsPath)
+
+  def test_getStepsForProblem(self):
+    problem1 = "5x=10"
+    problem2 = "3x+2x=5*2"
+    sol1 = ['5x=10', '(5x) / 5 = 10/5', 'x = 10/5', 'x = 2']
+    sol2 = ['3x+2x=5*2', '(3 + 2)x = 5 * 2', '5x = 5 * 2', '5x = 10', '(5x) / 5 = 10/5', 'x = 10/5', 'x = 2']
+    steps = mathstepsWrapper.getStepsForProblem(problem1)
```

Comment:
The assertion fails for me at this step for some reason. I have changed my prodConfig, installed mathsteps via npm as per @RyanYoung25  instructions.

```
`/home/micah/seniorDesign/mathsteps/node_modules/mathsteps/lib/simplifyExpression/index.js:6
function simplifyExpressionString(expressionString, debug=false) {
                                                         ^

SyntaxError: Unexpected token =
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/micah/seniorDesign/mathsteps/node_modules/mathsteps/index.js:1:90)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
['5x=10']`
```

I'm using node version 4.2.6 and npm version 3.5.2. Help on updating would be appreciated